src/subHeader/SubHeader.tsx

```diff
@@ -3,9 +3,10 @@ import React from 'react'
 import { StyledSubHeader } from './SubHeader.style'
 
 export type SubHeaderProps = Readonly<{
-  children: string
+  children: React.ReactNode
+  as?: keyof JSX.IntrinsicElements | React.ComponentType<any> // from styled-components types
```

Comment:
this is native on styled-component we should not use it here

Reply:
Didn't work as it's wrapping another styled component. That's why they put the forwardedAs but I couldn't make it work. Any idea?

Reply:
I remember I tried but couldn't use it out of the box
![Screen Shot 2021-03-11 at 17 20 14](https://user-images.githubusercontent.com/373381/110819077-2143f200-828e-11eb-888a-dc22b0e0cfa4.png)